components/movementsensor/adxl345/adxl345.go

```diff
@@ -232,6 +358,44 @@ func (adxl *adxl345) writeByte(ctx context.Context, register, value byte) error
 	return handle.WriteByteData(ctx, register, value)
 }
 
+func (adxl *adxl345) configureInterruptRegisters(ctx context.Context) error {
+	if adxl.interruptsEnabled == 0 {
+		return nil
+	}
+	adxl.configuredRegisterValues[IntEnableAddr] = adxl.interruptsEnabled
+	adxl.configuredRegisterValues[IntMapAddr] = adxl.interruptsMap
+	for key, value := range defaultRegisterValues {
+		if configuredVal, ok := adxl.configuredRegisterValues[key]; ok {
+			value = configuredVal
+		}
+		err := adxl.writeByte(ctx, key, value)
+		if err != nil {
```

Comment:
Mild preference for rolling these two lines together like my suggestion from earlier in this file:
```go
if err := adxl.writeByte(ctx, key, value); err != nil {
    ...
}
```